docs/docsite/rst/roadmap/COLLECTIONS_12.rst

```diff
@@ -0,0 +1,92 @@
+..
+   THIS DOCUMENT IS OWNED BY THE ANSIBLE COMMUNITY STEERING COMMITTEE. ALL CHANGES MUST BE APPROVED BY THE STEERING COMMITTEE!
+   For small changes (fixing typos, language errors, etc.) create a PR and ping @ansible/steering-committee.
+   For other changes, create a discussion as described in https://github.com/ansible-community/community-topics/blob/main/community_topics_workflow.md#creating-a-topic
+   to discuss the changes.
+   (Creating a draft PR for this file and mentioning it in the community topic is also OK.)
+
+.. _ansible_12_roadmap:
+
+====================
+Ansible project 12.0
+====================
+
+This release schedule includes dates for the `ansible <https://pypi.org/project/ansible/>`_ package, with a few dates for the `ansible-core <https://pypi.org/project/ansible-core/>`_ package as well. All dates are subject to change. See the `ansible-core 2.19 Roadmap <https://docs.ansible.com/ansible-core/devel/roadmap/ROADMAP_2_19.html>`_ for the most recent updates on ``ansible-core``.
+
+.. contents::
+   :local:
+
+
+Release schedule
+=================
+
+
+:2025-03-31: ansible-core feature freeze, stable-2.19 branch created.
+:2025-04-07: Start of ansible-core 2.19 betas
+:2025-04-08: Ansible-12.0.0 alpha1 [1]_
+:2025-04-28: First ansible-core 2.19 release candidate.
+:2025-04-29: Ansible-12.0.0 alpha2 [1]_
+:2025-05-19: Ansible-core-2.19.0 released.
+:2025-05-19: Last day for collections to make backwards incompatible releases that will be accepted into Ansible-12. This includes adding new collections to Ansible 12.0.0; from now on new collections have to wait for 12.1.0 or later.
+:2025-05-20: Ansible-12.0.0 beta1 -- feature freeze [2]_ (weekly beta releases; collection owners and interested users should test for bugs).
```

Comment:
Since the next week's release is planned to be the release candidate, I think we can remove the sentence about 'weekly beta releases':
```suggestion
:2025-05-20: Ansible-12.0.0 beta1 -- feature freeze [2]_ (collection owners and interested users should test for bugs).
```